components/authentication/signup/sharedPrimitives/AuthBtnLink.tsx

```diff
@@ -20,3 +21,29 @@ const AuthBtnLink = (
 };
 
 export default AuthBtnLink;
+
+export const SupportLinks = (action: string) => {
+  return (
+    <>
+      <div className="signInSupportLinks" data-cy="signInSupportLinks">
+        <b>Can&apos;t access your account? </b>
+        <a
+          href="https://tis-support.hee.nhs.uk/trainees/support-faq/"
+          target="_blank"
+          rel="noreferrer"
+        >
+          Please read our FAQ&apos;s
+        </a>
+        <br />
+        <b>Still having issues? </b>
+        <a
+          className="supportEmail"
+          title="Please provide your GMC no. and brief description of the issue include screenshots if necessary"
+          href={`mailto:tis.support@hee.nhs.uk?subject=TSS Tech Support Query (GMC No.: <your GMC number>)&body=<Please provide brief description of the issue include screenshots if necessary>%0A%0AReferred Page:%0A${action}%0A%0ABrowser and OS Info:%0A${getUserAgentInfo()}%0A%0A`}
```

Comment:
This sentence could probably do with some punctuation. 
e.g. "Please provide a brief description of the issue. Include screenshots if necessary." OR
e.g. "Please...(include screenshots if necessary)."